tests/FamilyHubs.ServiceDirectory.Api.AcceptanceTests/Tests/Steps/ServiceSearchSteps.cs

```diff
@@ -11,16 +11,15 @@ namespace FamilyHubs.ServiceDirectory.Api.AcceptanceTests.Tests.Steps;
 /// </summary>
 public class ServiceSearchSteps
 {
-  private readonly ConfigModel _config;  
   private readonly string _baseUrl;
   private ServiceSearchRequest _request;
   private HttpResponseMessage _lastResponse;
   private HttpStatusCode _statusCode;
   private const string serviceSearchPath = "api/metrics/service-search";
   public ServiceSearchSteps()
   {
-      _config = ConfigAccessor.GetApplicationConfiguration();
-      _baseUrl = _config.BaseUrl;
+      ConfigModel config = ConfigAccessor.GetApplicationConfiguration();
+      _baseUrl = config.BaseUrl;
```

Comment:
I would recommend inlining this since it's not being used for other variables, i.e., `_baseUrl = ConfigAccessor.GetApplicationConfiguration().BaseUrl;`, that way you don't need to declare the config variable above :)